tweet_recommendations/other_methods/graph_summarization_for_hashtag_recommendation.py

```diff
@@ -212,18 +210,22 @@ def _random_walk(self, similarities: np.ndarray) -> np.ndarray:
         query_hashtag_index = np.argmax(similarities)
         preference_vector = np.zeros((len(self._hashtag_labels, )))[..., np.newaxis]
         preference_vector[query_hashtag_index] = 1
-        preference_vector = sps.csr_matrix(preference_vector, shape=preference_vector.shape, dtype=np.float32)
+        preference_vector = sps.csr_matrix(preference_vector,
+                                           shape=preference_vector.shape,
+                                           dtype=np.float32)
         similarity_rank_vertices = preference_vector
         nb_iteration = 0
         while True:
             previous_similarity_rank_vertices = similarity_rank_vertices
             if self.verbose:
                 print("Step: {}".format(nb_iteration + 1))
 
-            similarity_rank_vertices = self.damping_factor * self._transition_matrix.dot(similarity_rank_vertices) + (
-                    1 - self.damping_factor) * preference_vector
+            similarity_rank_vertices = self.damping_factor * self._transition_matrix.dot(
+                similarity_rank_vertices) + (
+                                               1 - self.damping_factor) * preference_vector
```

Comment:
Coś tu jest mocno nie tak z formatowaniem

Reply:
Weź pod uwagę, że ten branch nie odpowiada za tę klasę. W odpowiednim branchu nie ma czegoś takiego.

Reply:
OK